HHS/Head-Start-TTADP
# [TTAHUB-2036] Add TR delete and edit

## Description of change

Add ability for admin to delete or edit TR.

**DELETE:** If a event is suspended or not started it can be deleted by an ADMIN.
**EDIT:** If an even is not completed can change the title and creator (IST).

## How to test

- Make sure you can delete (only as admin) if the report is not started or suspended
- Make sure you can change the title and creator (only as admin)
- Make sure you can still take a TR through the entire process (completed session reports etc.)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2036


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
